go.mod

```diff
@@ -22,14 +22,14 @@ go 1.20
 
 require (
 	github.com/GoogleCloudPlatform/spark-on-k8s-operator v0.0.0-20201215015655-2e8b733f5ad0
-	github.com/apache/yunikorn-core v0.0.0-20230626064933-148f221ab4f1
-	github.com/apache/yunikorn-scheduler-interface v0.0.0-20230621102204-bcadd461d275
+	github.com/apache/yunikorn-core v0.0.0-20230801161208-759e14da2966
+	github.com/apache/yunikorn-scheduler-interface v0.0.0-20230731151735-8931af4dda26
```

Comment:
@FrankYang0529 could you update this to the latest core/SI?

Reply:
Hi @pbacsko, thanks for the review. The error in e2e test needs to update yunikorn-core. I will send a PR to yunikorn-core first and then back to update to the latest version here.

https://issues.apache.org/jira/browse/YUNIKORN-1910